external_push/models.py

```diff
@@ -402,57 +416,126 @@ class Meta:
     # trigger_next_at = models.DateTimeField(default=timezone.now, help_text="When to next trigger a push")
 
     def __str__(self):
-        return self.gravity_sensor_to_push.name
+        if self.device_type == "gravity":
+            return self.gravity_sensor_to_push.name
+        
+        return self.brewpi_to_push.device_name
 
     def data_to_push(self):
         # For Brewfather, we're just cascading a single gravity sensor downstream to the app
-        to_send = {'report_source': "Fermentrack", 'name': self.gravity_sensor_to_push.name}
-
+        to_send = {'report_source': "Fermentrack"}
+ 
         # TODO - Add beer name to what is pushed
 
-        latest_log_point = self.gravity_sensor_to_push.retrieve_latest_point()
+        #{
```

Comment:
I haven't yet reviewed this block of code properly yet. It's hard for me to envision the context on GitHub so I want to clone the branch and review it that way. Give me a bit!